web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -62,6 +63,16 @@ const NavigationContainer = styled.div`
     0px 1px 1px rgba(0, 0, 0, 0.14), 0px 1px 3px rgba(0, 0, 0, 0.12);
 `;
 
+const PoweredByLogo = styled.div`
```

Comment:
I was wondering why we use `div` for logos. It seems that the more semantically correct option is `img`:
https://stackoverflow.com/a/1469139
https://stackoverflow.com/a/492834